Assets/MixedRealityToolkit-Examples/StandardShader/Scenes/MaterialGallery.unity

```diff
@@ -0,0 +1,5793 @@
+%YAML 1.1
```

Comment:
Seen needs to follow the new standards for examples and have a DialogBox detailing the reason for the scene.  Check @cre8ivepark work in the updated examples

Reply:
Ah good to know! I'll look @cre8ivepark's scene and make that change.

Reply:
@Cameron-Micka Sorry, I forgot to talk about it. You can search SceneDescriptionPanel prefab and include one in the scene. It does not have to be through - could be a short summary. You can find it in other examples scene too.

Reply:
@cre8ivepark awesome! I will be sure to use that prefab.

Reply:
@DDReaper Are there any other changes you would like me to make? I believe I've addressed all of your review comments.